app/cypress/integration/cif-external/index.spec.js

```diff
@@ -0,0 +1,27 @@
+describe("the external projects page", () => {
```

Comment:
Something to discuss with the whole team--how much e2e testing do we want to do at this stage? A lot of the first slice we already know will be temporary, so might be worth keeping the e2e testing light

Reply:
Good point! Let me share this in the devs channel.